<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>566</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>566</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="Products">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Products</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="add_product">
        <property name="text">
         <string>Add product</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_client">
        <property name="text">
         <string>Add client</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_provider">
        <property name="text">
         <string>Add provider</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="write_into_file">
        <property name="text">
         <string>Write into file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_product">
        <property name="text">
         <string>Delete product</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_client">
        <property name="text">
         <string>Delete client</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_provider">
        <property name="text">
         <string>Delete provider</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="read_from_file">
        <property name="text">
         <string>Read from file</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="Clients">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Clients</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Providers">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Providers</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTableView" name="table_products"/>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="table_clients">
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QTableView" name="table_providers"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
